clippy_utils/src/lib.rs

```diff
@@ -15,6 +15,7 @@
     clippy::missing_errors_doc,
     clippy::missing_panics_doc,
     clippy::must_use_candidate,
+    clippy::redundant_repeat_take,
```

Comment:
I’m unsure whether I should add an allow rule like this, or if I should fix the part of the code that triggers the linting error, since currently there is only [one part](https://github.com/rust-lang/rust-clippy/blob/b3fadd5a8ef45bcd24e9dc31d32926cb7ad26444/clippy_utils/src/lib.rs#L3417) that is affected by this lint...

Reply:
This lint looks useful to have to me so I think keeping it enabled is a good idea. The case also looks like it should be fairly easy to fix